extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -8,6 +8,7 @@
 @import 'extensions/wikia/ThemeDesigner/css/ThemeTab';
 @import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index.scss';
 @import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_buttons.scss';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_dropdowns.scss';
```

Comment:
I don't think you need to import this file. Design System compiled (`.css`) styles are included with the Design System extension itself. Two `.scss` files in the lines above are imported, because this way it's possible to use `.scss` variables and extend `.scss` rules (for buttons in this case).